### ISSUE TITLE
How to warn PSC members for new requests ?

### ISSUE THREAD

**Issue by rousseldenis** [OCA/repo-maintainer-conf#20]
As I said in the mailing list at the creation of this (with no answer), we should find a way to warn PSC members for new appliances.

@etobella

**Comment by pniederlag** [comment]
is this still relevant?

**Comment by sbidoul** [comment]
Yes. But before working on this, we need to clarify the scope of PSC responsibilities.
